cmd/kubelet/app/server.go

```diff
@@ -768,7 +773,7 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 
 	if kubeDeps.ContainerManager == nil {
 		if s.CgroupsPerQOS && s.CgroupRoot == "" {
-			klog.InfoS("--cgroups-per-qos enabled, but --cgroup-root was not specified.  defaulting to /")
+			logger.Info("--cgroups-per-qos enabled, but --cgroup-root was not specified.  defaulting to /")
```

Comment:
```suggestion
			logger.Info("--cgroups-per-qos enabled, but --cgroup-root was not specified.  Defaulting to /")
```

Reply:
Thank you for your review.
I fixed here https://github.com/kubernetes/kubernetes/pull/127887/commits/2ac71d903967e1f1a3eab6b473a9c9cc0a0f02a8 👍